src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public abstract class Figure {
```

Comment:
ти можеш імплементити інтерфейс тут і тобі не доведеться це робити для кожного класу окремо 
```suggestion
public abstract class Figure implements Figures {
```